auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneNumberVerificationHandler.java

```diff
@@ -60,4 +63,14 @@ public void submitVerificationCode(String number, String code) {
                 PhoneAuthProvider.getCredential(mVerificationId, code),
                 false)));
     }
+
+    public void onSaveInstanceState(@NonNull Bundle outState) {
+        outState.putString(VERIFICATION_ID_KEY, mVerificationId);
```

Comment:
I looked around, but couldn't find sample code with `onSaveInstanceState`. 😕 So I guess it's time to patent my new invention, right? 😂 

I wouldn't mind moving it to the activity, but it made more sense to me to let the ViewModel manage its saved state. (Otherwise, we'd end up with getters and setters for `mVerificationId` which shouldn't be used outside of the state restoration context.) Whad'ya think?

Reply:
I am cool with it, let's just pinky swear not to show Jake Wharton or Yigit the monster we've created!

Reply:
👌😂